examples/traffic-routing-experiment-step/rollout-alb-experiment-step.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
```

Comment:
Im just learning about `Experiments` while reading this PR. Looking at this examples folder I don't see the `Experiment` resource defined in the yamls. Isn't the `experiment-alb` and `experiment-smi` experiment yamls necessary so user could have a complete runnable example?

Reply:
@leoluz No, the `Experiment` here is created based on the information provided in the Rollout Experiment step. More information here: https://argoproj.github.io/argo-rollouts/features/experiment/#integration-with-rollouts